doc/man3/OSSL_trace_set_channel.pod

```diff
@@ -46,7 +53,8 @@ OSSL_trace_set_callback() instead.
 OSSL_trace_set_callback() is used to enable the given trace
 C<category> by giving it the tracer callback C<cb> with the associated
 data C<data>, which will simply be passed through to C<cb> whenever
-it's called.
+it's called. The callback function is internally wrapped by a
+dedicated BIO object, the so called callback trace channel.
```

Comment:
Since the term is being defined, would it make sense to italicise _callback trace channel_?